public/css/style.css

```diff
@@ -1,3 +1,8 @@
+a, a:hover, a:visited, a:active {
```

Comment:
We probably don't want to change all <a> tag attributes. It removes styling for links on the signup page and login page. Probably best to give the <a> tag on the home page an id attribute and change the css for it.